src/bin/pg_autoctl/keeper_pg_init.c

```diff
@@ -118,6 +312,15 @@ keeper_pg_init_and_register(Keeper *keeper, KeeperConfig *config)
 	bool postgresInstanceExists = pg_setup_pgdata_exists(&pgSetup);
 	bool postgresInstanceIsPrimary = pg_setup_is_primary(&pgSetup);
 
+	if (postgresInstanceExists)
+	{
+		if (!keeper_ensure_pg_configuration_files_in_pgdata(config))
+		{
+			log_fatal("Existing postgresql instance is not supported");
```

Comment:
We need to give more information to the user here I believe. Just saying that we only support core Postgres setup where the configuration files live in PGDATA might be enough. But "not supported" is not actionable input to the user.